docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -251,6 +252,19 @@ export AZURE_ACCOUNT_ACCESS_KEY='NN1KHxKKkj20ZO92EMiDQjx3wp2kZG4UUvfAGlgGWRn6sPR
 ```
 ````
 
+## Huggingface Datasets
+
+To authenticate with uploading and for private downloads, set the `HF_TOKEN` beforehand.
+
+The format of the urls can be found in [HF's documentation](https://huggingface.co/docs/huggingface_hub/guides/hf_file_system), for example:
```

Comment:
How about this:

"Set the Huggingface token in the run environment as shown below

````{tabs}
```{code-tab} py
import os
os.environ['HF_TOKEN'] = 'EXAMPLEFODNN7EXAMPLE'
```

```{code-tab} sh
export HF_TOKEN='EXAMPLEFODNN7EXAMPLE'
```
````
"

You can remove the line 260-263.